sos/cleaner/mappings/__init__.py

```diff
@@ -28,11 +29,33 @@ class SoSMap():
     ignore_short_items = False
     match_full_words_only = False
 
-    def __init__(self):
-        self.dataset = {}
+    def __init__(self, workdir='/tmp'):
+        self.dataset = {}  # TODO: mention the child classes can update it as
+        # well as add_sanitised_item_to_dataset method, so the files dont need
```

Comment:
Perhaps it may be better to add this comment on top of self.dataset = {} instead of starting at the end of the line and then continuing below?